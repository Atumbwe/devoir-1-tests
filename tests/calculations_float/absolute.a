8.6 abs