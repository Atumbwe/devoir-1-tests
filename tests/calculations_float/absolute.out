 _____
< 8.6 >
 -----
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
